config/jobs/kubernetes-client/haskell/client-haskell-presubmits.yml

```diff
@@ -0,0 +1,20 @@
+presubmits:
+  kubernetes-client/haskell:
+  - name: client-haskell-unit-tests
+    always_run: true
+    decorate: true
+    spec:
+      containers:
+      - image: ubuntu:bionic
+        env:
+        command:
+        - bash
+        args:
+        - -c
+        - |
+          set -euo pipefail
```

Comment:
You may have an easier time if you put this code in a bash file in this repository, and call that script. `decorate: true` will [pull down a repository for you.](https://github.com/kubernetes/test-infra/blob/master/prow/pod-utilities.md)

Also, I see that this script is just calling the script at `https://get.haskellstack.org/`. Is _that_ script stored in git somewhere?

Reply:
I see that some projects keep their `presubmits.sh` in the project repo, like [repo-infra](https://github.com/kubernetes/repo-infra/blob/master/presubmit.sh). Do you recommend we keep a script like this, or should I put a script somewhere in this (test-infra) repository? If its the later, can you please tell me where I should keep the script?

About the script at `https://get.haskellstack.org/`, it is in a git repository, but I guess it would be better to have `stack` (the build tool) in the base image for running tests.

Reply:
A script for another repo's presubmits should be with the code it's testing, in that repository. Otherwise, you'll have a harder time maintaining those tests.